### PR TITLE
CHANGELOG: update 3.2 changelog and 3.3 upgrade document for #11691

### PR DESCRIPTION
**Pull request by wswcfan**
RT

### CODE REVIEW

**Comment by wswcfan** [comment]
PTAL? Thanks. @jingyih @mitake

File: Documentation/upgrades/upgrade_3_3.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Before [starting an upgrade](#upgrade-procedure), read through the rest of this
...
 
-**NOTE:** if you enable auth and use lease(lease ttl is small),it has a high probability to encounter [issue](https://github.com/etcd-io/etcd/issues/11689) that will result in data inconsistency. It is strongly recommended to upgrade to the latest version of 3.2 to fix this problem, and then upgrade to 3.3. The latest version is not yet available, please pay attention to [changelog](https://github.com/etcd-io/etcd/blob/master/CHANGELOG-3.2.md).
+**NOTE:** if you enable auth and use lease(lease ttl is small), it has a high probability to encounter [issue](https://github.com/etcd-io/etcd/issues/11689) that will result in data inconsistency. It is strongly recommended upgrading to 3.2.31+ firstly to fix this problem, and then upgrade to 3.3. In addition, if the user without permission sends a LeaseRevoke request to the 3.3 node during the upgrade process, it may still cause data corruption, so it is best to ensure that your environment doesn't exist such abnormal calls before upgrading, see [#11691](https://github.com/etcd-io/etcd/pull/11691) for detail.
```
Comment (spzala) [inline Documentation/upgrades/upgrade_3_3.md:15]: @wswcfan nice thanks! Meanwhile you wain on other reviewers, a nit - considering LeaseRevoke a method you should highlight it as `LeaseRevoke`

Comment (wswcfan) [reply Documentation/upgrades/upgrade_3_3.md]: Thanks for your suggestion, I have highlight it now.

**Comment by spzala** [comment]
@wswcfan thanks for addressing my suggestion. 
lgtm

**Pull request merged**